test/fail_compilation/ice11790.d

```diff
@@ -1,7 +1,7 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/ice11790.d(8): Error: cannot pass type `string` as a function argument
```

Comment:
You could also extend this test with the `typeof` line (and an appropriate bugzilla link) since they target the same code.
(Less file bloat in the test suite)

Reply:
Seems like the wrong place to put it since it's not an ICE, but I'll see if there's another fail_compilation test that's a better home

Reply:
Agreed, yould could also remove this in favour of your file (while keeping link to the original issue).
Just don't have multiple equivalent tests.